### DISCUSSION TITLE
Automatic Reshuffling of the random_tracks Playlist

### DISCUSSION THREAD

**Discussion by qdrop17** [music-assistant/support#3053] [Q&A] [answered]
I love how this software integrates my favorite music services. Could we implement a feature to automatically reshuffle the random_tracks playlist daily?

If a native solution isn't possible, perhaps a cron job could work as an alternative.

https://github.com/music-assistant/server/blob/433bc0b99fb69ff5844a77c1ed2487add37995d2/music_assistant/server/providers/builtin/__init__.py#L544

Thoughts on implementing this?

This discussion here is probably related: https://github.com/orgs/music-assistant/discussions/2403

**Comment by marcelveldt** [comment, marked as answer]
It is already auto shuffled on every sync interval.
